build_all.sh

```diff
@@ -0,0 +1,22 @@
+#! /bin/sh
+
+# build development version of the compiler; can be rerun safely
+
+set -u
```

Comment:
Add a comment what this command does. And ensure it's in Posix.

Reply:
done; set -e, set -u are standard posix stuff